apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -179,7 +179,7 @@ int tc_kernel_main(int argc, char *argv[])
 #endif
 
 #ifdef CONFIG_TC_KERNEL_TIMER
-	timer_main();
+	timer_tc_main();
```

Comment:
Why this change only for timer test, if so we should make the convention same for all the tc's

Reply:
Because it has too common name, it easily causes the same problem we think. That's why we modify it together. Another tc also needs to be modified later.